### PR TITLE
:running: enable lint goconst

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Enable lint goconst check
Fix issues (clean code)

### CODE REVIEW

File: testdata/project-v2/Makefile [hunk 1]
```diff
@@ -52,7 +52,7 @@ vet:
 
 # Generate code
 generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```
Comment (Adirio) [inline testdata/project-v2/Makefile]: Fixed and tracked by #1288.

**Comment by Adirio** [comment]
LGTM after #1288 gets merged.

**Comment by mengqiy** [comment]
@camilamacedo86 Rebase needed for this one too

**Comment by camilamacedo86** [comment]
Hi @mengqiy it is missing the approval to be merged :-)

**Pull request merged**